components/gfx/display_list/mod.rs

```diff
@@ -99,38 +176,26 @@ impl DisplayList {
         }
     }
 
-    fn hit_test_scroll_root<'a>(&self,
-                                traversal: &mut DisplayListTraversal<'a>,
-                                scroll_root: &ScrollRoot,
-                                mut translated_point: Point2D<Au>,
-                                client_point: &Point2D<Au>,
-                                scroll_offsets: &ScrollOffsetMap,
-                                result: &mut Vec<DisplayItemMetadata>) {
-        // Adjust the translated point to account for the scroll offset if
-        // necessary. This can only happen when WebRender is in use.
-        //
-        // We don't perform this adjustment on the root stacking context because
-        // the DOM-side code has already translated the point for us (e.g. in
-        // `Window::hit_test_query()`) by now.
-        if let Some(scroll_offset) = scroll_offsets.get(&scroll_root.id) {
-            translated_point.x -= Au::from_f32_px(scroll_offset.x);
-            translated_point.y -= Au::from_f32_px(scroll_offset.y);
-        }
-        self.hit_test_contents(traversal, &translated_point, client_point, scroll_offsets, result);
-    }
-
     fn hit_test_stacking_context<'a>(&self,
```

Comment:
Could you add a comment explaining that this function translates the point to account for the scroll offset?